src/display/process/procGraphs.go

```diff
@@ -30,21 +31,21 @@ import (
 var runProc = true
 
 func getChildProcs(proc *process.Process) []string {
-	childProcs := []string{"PID                   Command"}
+	headerString := "PID" + strings.Repeat(" ", 19) + "Command"
+	childProcs := []string{headerString}
 	for _, proc := range proc.Children {
+		var processData, spacesForCommandRowData string
+		processPid := strconv.Itoa(int(proc.Pid))
+		// 22 reflects position where row data for "Command" column should start (headerString has 19 spaces + length of ("PID") is 3 i.e. 22)
+		spacesForCommandRowData = strings.Repeat(" ", 22-len(processPid))
 		exe, err := proc.Exe()
 		if err == nil {
-			temp := strconv.Itoa(int(proc.Pid))
-			for i := 0; i < 22-len(strconv.Itoa(int(proc.Pid))); i++ {
-				temp = temp + " "
-			}
-			temp = temp + "[" + exe + "](fg:green)"
-			childProcs = append(childProcs, temp)
+			processData = processPid + spacesForCommandRowData + "[" + exe + "](fg:green)"
```

Comment:
The PID should be made `yellow` in case there is no error, similar to how it's done in the `else` clause.

Reply:
So in both cases it should be yellow ? Because the implementation I changed had yellow only for else case.

Reply:
Yes, it should be yellow uniformly! 😄

Reply:
Sure :+1: , updated the PR